cpp/benchmarks/fixture/nvbench_main.cpp

```diff
@@ -21,20 +21,22 @@
 
 #include <vector>
 
-// strip off the rmm_mode parameter before passing the
+// strip off the rmm_mode and cuio_host_mem parameters before passing the
 // remaining arguments to nvbench::option_parser
 #undef NVBENCH_MAIN_PARSE
-#define NVBENCH_MAIN_PARSE(argc, argv)         \
-  nvbench::option_parser parser;               \
-  std::vector<std::string> m_args;             \
-  for (int i = 0; i < argc; ++i) {             \
-    std::string arg = argv[i];                 \
-    if (arg == cudf::detail::rmm_mode_param) { \
-      i += 2;                                  \
-    } else {                                   \
-      m_args.push_back(arg);                   \
-    }                                          \
-  }                                            \
+#define NVBENCH_MAIN_PARSE(argc, argv)                     \
+  nvbench::option_parser parser;                           \
+  std::vector<std::string> m_args;                         \
+  for (int i = 0; i < argc; ++i) {                         \
```

Comment:
This can be for a separate PR, but in cuDF tests (and other RAPIDs tests) we use the cxxopts library to parse command line arguments. Might be good to use it here too.  See also https://github.com/NVIDIA/nvbench/issues/87